packages/fastify-renderer/package.json

```diff
@@ -68,8 +68,10 @@
     "path-to-regexp": "^6.2.1",
     "sanitize-filename": "^1.6.3",
     "stream-template": "^0.0.10",
+    "ts-node": "^10.9.1",
```

Comment:
Necessary for loading TS workers

Reply:
Can we make this a dev dependency? And ensure that when this package is built, those workers are transpiled down to JS?

Reply:
How could we detect if its being called from a built package or not? Does the `devMode` flag control `NODE_ENV` or can I still use it?

Reply:
I think that the only time that TS workers are really relevant is when this package is being developed -- not when the end application is being developed. `.devMode` will be true when this package is built and assembled and downloaded from NPM, but a dev is working on say Gadget core on their local machine. So, I think it might be easier to just use an env var or something like that that is only set to true when this package is being developed to require `ts-node`?

Reply:
I'll give it some thought. I'll come up with something tomorrow.

Reply:
I think checking for the existence of `StaticWorker.js` and loading it if it is available would be a good way to approach it. No need for flags.

If it's not there, load `ts-node` and try to import `StaticWorker.ts`.